src/main/java/bio/terra/stairway/FlightContext.java

```diff
@@ -194,4 +194,19 @@ public String toString() {
         .append("flightStatus", flightStatus)
         .toString();
   }
+
+  public String toStringTrunc() {
```

Comment:
A comment here indicating which parameter is missing would be useful. If you are not planning to truncate any strings (as suggested by @nmalfroy below), then I would rename the function to `toStringQuiet` or `toStringTerse` or something along those lines.